### DISCUSSION TITLE
Copying the theme and source code of https://pages.github.com/

### DISCUSSION THREAD

**Discussion by JKSH** [community/community#22297] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I really like how <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a> has been implemented. In particular, I like how specific content is shown/hidden based on the user’s selections. The fade-in effect is a nice touch too.</p>
<p>I’d like to create a tutorial for my own project with a similar look-and-feel (to be hosted on GitHub Pages). I can see that <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a> was built with Jekyll, using just one CSS file and one custom JS script + jQuery.</p>
<p>My questions are:</p>
<ol>
<li>Is the source code or theme for <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a> available for public use?</li>
<li>Am I allowed to copy and modify <a href="https://pages.github.com/css/pages.css" rel="nofollow noopener">https://pages.github.com/css/pages.css</a> and <a href="https://pages.github.com/js/application.js" rel="nofollow noopener">https://pages.github.com/js/application.js</a> for my own project?</li>
</ol>
<p>Thanks!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jksh">@jksh</a>,</p>
<p>Thanks for being here! The source code for our pages site is not public.  There are loads of other <a href="https://jekyllrb.com/showcase/" rel="nofollow noopener">pages</a> using Jekyll that can help you create a similar project.</p>
<p>Best,</p>
<p>Andrea</p>
